DistributionPackages/Neos.NeosIo.ServiceOfferings/Resources/Private/Fusion/Content/AwardReference.fusion

```diff
@@ -1,8 +1,20 @@
 prototype(Neos.NeosIo.ServiceOfferings:Content.AwardReference) < prototype(Neos.Neos:ContentComponent) {
     award = ${q(node).property('award')}
     title = ${q(node).property('title')}
+    projectUrl = ${q(node).property('projectUrl')}
 
-    renderer = afx`
-        <li title={props.title}><i class="fa fa-award"></i> {props.award.label} ({props.title})</li>
-    `
+    renderer = Neos.Fusion:Case {
+        hasLink {
+            condition = ${q(node).property('projectUrl')}
+            renderer = afx`
+                <li title={props.title}><i class="fa fa-award"></i> {props.award.label} (<a href={props.projectUrl} target="_blank">{props.title}</a>)</li>
```

Comment:
you could as well add the condition here: 
`<a href={props.projectUrl} @if.hasProjectUrl={props.projectUrl}>`
and then add 
`<span @if.hasNoProjectUrl={!props.projectUrl}>` as alternative.

Reply:
Thanks for the hint! @regniets :)